android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -919,11 +919,18 @@ public boolean onRenderProcessGone(WebView view, RenderProcessGoneDetail detail)
           Log.w("RNCWebViewManager", "The WebView rendering process was killed by the system.");
         }
 
-        WritableMap event = Arguments.createMap();
-        event.putBoolean("didCrash", detail.didCrash());
+        // if webView is null, we cannot return any event
+        // since the view is already dead/disposed
+        // still prevent the app crash by returning true.
+        if(webView == null){
+          return true;
+        }
+
+        WritableMap event = createWebViewEvent(webView, webView.getUrl());
+
         dispatchEvent(
-          view,
```

Comment:
Just in case my other comment got lost since it was labeled "outdated," please bring back `didCrash`

Reply:
I'm sorry, I did see your comment and I thought I had kept the `didCrash` value, but I didn't.  Adding it back now.

Reply:
Should be done now.

Reply:
Thanks! Looks good